pkgs/applications/graphics/apitrace/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "apitrace-${version}";
-  version = "7.1";
+  version = "7.1-git-20170411";
```

Comment:
It's even better to use `pre-20170411`, since then it will preceed version `7.1` in the lexicographical ordering used by `nix-env -u` (i.e. `7.1-pre-20170411` will upgrade to `7.1`, whereas `7.1-unstable-...` will not).